lib/models/index.ts

```diff
@@ -29,11 +29,13 @@ export interface ModelsProps {
 export class Models extends Construct {
   public readonly models: SageMakerModelEndpoint[];
   public readonly modelsParameter: ssm.StringParameter;
+  public readonly bedrockEnabledModelsParameter: ssm.StringParameter;
 
   constructor(scope: Construct, id: string, props: ModelsProps) {
     super(scope, id);
 
     const models: SageMakerModelEndpoint[] = [];
+    const bedrockEnabledModels: string[] = ['anthropic.claude-3-haiku-20240307-v1:0', 'anthropic.claude-3-sonnet-20240229-v1:0'];
```

Comment:
This value should come from the `config.json` file. You need to add a new property to the `SystemConfig` interface to store these values and add a multiselect option to the magic config to let the user choose which models to enable.

Reply:
Should we add a free text with comma seperated model ids or a multi-selected to select from all the available foundation models ? I would vote for free text, since it doesn't need maintance - adding new models - and a bit more user-friendly - user will type instead of scrolling through the models to select the one's needed.

Reply:
I you just want to allow-list few models, a text field could work fine, but you would still need to validate the modelIds. I think a multiselect is more user friendly and you can always populate the values dynamically by querying the bedrock API. Anyway, to add new models we also need to add them to the rest of the code.